WcaOnRails/spec/models/user_spec.rb

```diff
@@ -65,16 +65,7 @@
     user.confirm
   end
 
-  it "allows demotion of a senior delegate with no subordinate delegates" do
-    senior_delegate = FactoryBot.create :senior_delegate
-
-    senior_delegate.delegate_status = ""
-    senior_delegate.region_id = nil
-    expect(senior_delegate.save).to eq true
-    expect(senior_delegate.reload.delegate_status).to eq nil
-  end
-
-  it "requires senior delegate be a senior delegate" do
```

Comment:
I think I forgot to rename this test earlier. This is actually replacement of the next test which is removed.

We don't need test for senior delegate to be a senior delegate because we anyway fetch senior delegates from roles with status senior_delegate. Also, currently it's in transition stage, so will have good tests for roles once the transition is completed.